lib/media/closed_caption_parser.js

```diff
@@ -88,6 +90,10 @@ shaka.media.ClosedCaptionParser = class {
    * @override
    */
   init(initSegment) {
+    shaka.log.debug('Passing new init segment to CEA parser');
+    // Reset underlying decoder when new init segment arrives
+    // to clear stored pts values.
+    this.reset();
```

Comment:
Will this cause undesirable behavior on a bit-rate change?

Reply:
Can you be more specific?  It seems to be the explicit intent of the author to reset CEA decoding when we switch bitrates (append init segment).  What undesirable behavior are you seeing / anticipating?

Reply:
i think they need to distinguish period change from bitrate change.

Reply:
Yes, you are right. @caridley can you send a PR to fix it?

Reply:
I don't think I will have time to do that. Sorry

Reply:
I have a fix for it, https://github.com/shaka-project/shaka-player/pull/6671